themes/proton-carbon/v1/theme.json

```diff
@@ -28,7 +28,7 @@
         "hover": "#24212B",
         "shade": "#7C5CFF12",
         "backdrop": "#35353580",
-        "textSelection": "#7C5CFF20"
+        "textSelection": "#7C5CFF"
```

Comment:
Can we keep the previous translucent color for textSelection? This one makes the selected text unreadable.

Reply:
```suggestion
        "textSelection": "#7C5CFF20"
```

Reply:
Translucent color could work if the textSelection is done on "paragraph": "#6D6A79" as this color is dull but the updated color   "paragraph": "#D3D3D3", is very bright. So textSelection will not show if  "textSelection": "#7C5CFF20" is used instead of the updated one. Please check this once again.